### PR TITLE
added visual blocks docs

### LINKED ISSUE #1234: Document visual block selector
The new visual selector is opt-in from the schema. It's now merged but not released; I'm creating this issue to track creating docs, but it's okay to release it before docs are ready (those that are interested can [reference the PR](https://github.com/tinacms/tinacms/pull/2563) to find implementation details for now). 

Potential places to update:
- [Blocks Field](https://tina.io/docs/reference/toolkit/fields/blocks/)
- [The 'object' field](https://tina.io/docs/reference/types/object/)

Things to add:
- API details & options
- An example of the visual selector 

Notes:
Good reference material https://tina.io/docs/editing/blocks/

### PR DESCRIPTION
**Pull request by logan-anderson**
closes #1234

TODO: add images (might be a separate PR that @spbyrne  can do next week.)

### CODE REVIEW

File: content/docs/editing/blocks.md [hunk 1]
```diff
@@ -217,3 +217,112 @@ In our case:
...
 - FieldName: "Blocks"
 - BlockTemplateName: "Content" | "Hero" | "Features"
+
+
+## Adding a visual block selector (Experimental)
+
+{{ WarningCallout text="This is an experimental feature, and the API is subject to change. Have any thoughts? Let us know in the chat, or through one of our [community channels](/community/)!" }}
+
+
+This visual block selector allows editors to select blocks from a a set images instead of text.
```
Comment (perkinsjr) [inline content/docs/editing/blocks.md:227]: Grammar.. You have "from a a set"

**Review by perkinsjr — CHANGES_REQUESTED** [review]

**Comment by spbyrne** [comment]
I should mention both `category` and `previewSrc` are optional; if a preview isn't provided the block will still be available with a title to click on, just no preview.

File: content/docs/editing/blocks.md (hunk 1 above)
Comment (logan-anderson) [reply content/docs/editing/blocks.md]: Thanks @perkinsjr it has been fixed

**Comment by logan-anderson** [comment]
> I should mention both `category` and `previewSrc` are optional; if a preview isn't provided the block will still be available with a title to click on, just no preview.

@spbyrne updated docs to reflect this.

**Review by perkinsjr — APPROVED** [review]

**Pull request merged**